warp10/src/main/java/io/warp10/script/MemoryWarpScriptStack.java

```diff
@@ -616,15 +616,15 @@ public void exec(String line) throws WarpScriptException {
             }
             multiline.append(stmt);
             continue;
-          } else if (WarpScriptStack.COMMENT_END.equals(stmt)) {
+          } else if (stmt.endsWith(WarpScriptStack.COMMENT_END)) {
```

Comment:
This will break existing WarpScript code in several ways.

Code which uses variables ending with '*/' will break and code which has any token ending in '*/' within a block comment will also break since this token will now be considered the end of the comment.

Reply:
Well github does not like the construct * followed by / it seems...